### PR TITLE
MTSRE team perms

### PR DESCRIPTION
**Pull request by sedroche**

### CODE REVIEW

File: deploy/backplane/mtsre/config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchLabels:
+      api.openshift.com/addon-managed-odh: "true"
```
Comment (sedroche) [inline deploy/backplane/mtsre/config.yaml:4]: I think this is correct, the addon id is `managed-odh` and the others are prefixed `addon-`

**Comment by wanghaoran1988** [comment]
@sedroche My understanding is "managed-odh" is one addon that managed by mtsre team, so I'd like the perms related to "managed-odh" can be in a separate directory similar with cssre team, see https://github.com/openshift/managed-cluster-config/tree/master/deploy/backplane/cssre/rhoam

**Comment by wanghaoran1988** [comment]
I am not sure whether we should enable them to escalate, but it looks like we should in long term, @cblecker WDYT? if the answer is yes, we should update https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/elevated-sre/10-impersonate-cluster-admin.ClusterRoleBinding.yml

File: deploy/backplane/mtsre/00-mtsre.namespace.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-backplane-mtsre
```
Comment (chopraapooja) [inline deploy/backplane/mtsre/00-mtsre.namespace.yml:4]: Why these files are prefixed with numbering ? is it apply order ?

**Comment by sedroche** [comment]
@cblecker Another question of whether this team will need monitoring perms like cssre. e.g. https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/cssre/01-cssre-monitoring-role.yml

**Comment by cblecker** [comment]
This seems fine to me

/approve

**Pull request merged**